### ISSUE TITLE
Extension icons should be 256x256 instead of 128x128

### ISSUE THREAD

**Issue by Code-DJ** [OmniSharp/omnisharp-vscode#5796]
Nit pick: all the extension icons are 128x128 instead of 256x256. Appear blurred on retina display on the extension details page in VS Code.

<img width="437" alt="image" src="https://github.com/dotnet/vscode-csharp/assets/8212554/215c5876-cb33-4f3f-9d5f-284212456e80">
<img width="461" alt="image" src="https://github.com/dotnet/vscode-csharp/assets/8212554/3dd2875a-0456-45ea-a544-6332fcf63712">
<img width="576" alt="image" src="https://github.com/dotnet/vscode-csharp/assets/8212554/f2edc01f-b0c6-4db2-bda2-e69681312a7b">

**Comment by timheuer** [comment]
> @timheuer is this something you'd like to look at (or know who should)?

Yep, keep it on me...I asked @Code-DJ to log it -- will work with design.
